src/lib/minions/types.ts

```diff
@@ -113,8 +113,9 @@ export interface KillableMonster {
 	effect?: (opts: {
 		messages: string[];
 		user: MUser;
+		tame?: Tame;
```

Comment:
Change to use `MTame`, so you can do `tame.totalLoot` instead of  `new Bank(tame.max_total_loot as ItemBank)`. Merge at your discretion when its ready.